app/views/info/_lead_metrics.html.erb

```diff
@@ -1,16 +1,33 @@
 <section id="lead-metrics">
   <% if lead_metrics %>
+    <%- if format == 'guide' %>
```

Comment:
I'm not keen on hard-coding the format here - `guide` isn't the only format that's multi-part (`programme` is also a multi-part format, see [an example here](https://www.preview.alphagov.co.uk/1619-bursary-fund)) and there could be other new formats introduced. I'd prefer a duck-typing approach (anything that's multi-part).